ios-nocodesign-11-1-dep-9-0-wo-armv7s-bitcode.cmake

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2017, Ruslan Baratov
+# All rights reserved.
+
+if(DEFINED POLLY_IOS_NOCODESIGN_11_0_DEP_9_0_BITCODE_CMAKE_)
+  return()
+else()
+  set(POLLY_IOS_NOCODESIGN_11_0_DEP_9_0_BITCODE_CMAKE_ 1)
+endif()
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_clear_environment_variables.cmake")
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_init.cmake")
+
+set(IOS_SDK_VERSION 11.1)
+set(IOS_DEPLOYMENT_SDK_VERSION 9.0)
+
+set(POLLY_XCODE_COMPILER "clang")
+polly_init(
+    "iOS ${IOS_SDK_VERSION} / Deployment ${IOS_DEPLOYMENT_SDK_VERSION} / Universal (iphoneos + iphonesimulator) / \
```

Comment:
Add list of libraries here.

https://github.com/ruslo/polly/blob/6b4b3d2b866fee571d900f9f5dfcb4fa4eab863f/ios-10-0-wo-armv7s.cmake#L19